packages/pigeon/platform_tests/shared_test_plugin_code/lib/example_app.dart

```diff
@@ -36,9 +34,11 @@ class _ExampleAppState extends State<ExampleApp> {
   }
 
   Future<void> initPlatformState() async {
-    api = AllVoidHostApi();
+    api = HostIntegrationCoreApi();
     try {
-      await api.doit();
```

Comment:
Is this a naming convention you would like changed throughout as well? There are a lot of doit methods in some generated files in this pr. Not that I am expecting them all to be changed here.

Reply:
I expect we won't need most of them long term and can just eliminate them during the consolidation. But also, for the integration tests in particular I'm trying to be more deliberate with the naming since we actually need to provide implementations for them, so having a name that describes what they are expected to do actually matters.

Most of the other files are just being generated/compiled, or sometimes unit tested, and the names there aren't as important. We can decide as we consolidate further what to call what we keep.